modules/apps/object/object-admin-rest-test/src/testIntegration/java/com/liferay/object/admin/rest/resource/v1_0/test/ObjectDefinitionResourceTest.java

```diff
@@ -141,6 +142,111 @@ public void tearDown() throws Exception {
 		}
 	}
 
+	@Test
+	public void testBatchRelatedSystemModifiableObjectDefinition()
```

Comment:
@JhossephG based on the pattern of this class your test should be defined in this `testPostObjectDefinition` test

Reply:
try to do something like that: `_testPostObjectDefinitionBatch`